schulcloud/Controllers/Dashboard/NewsOverviewViewController.swift

```diff
@@ -83,8 +83,12 @@ extension NewsOverviewViewController : NSFetchedResultsControllerDelegate {
     }
 }
 
-extension NewsOverviewViewController : ViewControllerHeightDataSource {
+extension NewsOverviewViewController : ViewHeightDataSource {
     var height : CGFloat {
         return tableView.contentSize.height + 20.0
     }
 }
+
+extension NewsOverviewViewController : PermissionInfoDataSource {
+    static var requiredPermission : UserPermissions { return UserPermissions.newsView }
```

Comment:
You don't have to use `var` to conform to the protocol. A shorter way to write this would be:
```swift
static let requiredPermissions = UserPermissions.newsView
```